test/kafka_test.go

```diff
@@ -490,3 +490,18 @@ func (s *CLITestSuite) TestKafkaClientQuotas() {
 		s.runIntegrationTest(test)
 	}
 }
+
+func (s *CLITestSuite) TestKafkaAutocomplete() {
+	tests := []CLITest{
+		{args: `__complete kafka cluster describe ""`, fixture: "kafka/describe-autocomplete.golden"},
+		{args: `__complete kafka link delete ""`, fixture: "kafka/link/list-link-delete-autocomplete.golden", useKafka: "lkc-describe-topic"}, // use delete since link has no describe subcommand
```

Comment:
Should we have a `link describe` command?

Reply:
We probably should since we have a list command. I'll make a ticket about this.